src/main/java/bio/terra/app/controller/ApiValidationExceptionHandler.java

```diff
@@ -20,6 +22,25 @@
 @ControllerAdvice
 public class ApiValidationExceptionHandler extends ResponseEntityExceptionHandler {
 
+    /**
+     * Since the default {@link ResponseEntityExceptionHandler#handleExceptionInternal}
+     * method is often passed a null body, we override it and substitute the exception's error message
+     * into an ErrorModel and return that in the response instead.
+     */
+    @Override
+    protected ResponseEntity<Object> handleExceptionInternal(
+        Exception ex, @Nullable Object body, HttpHeaders headers, HttpStatus status, WebRequest request) {
+
+        if (HttpStatus.INTERNAL_SERVER_ERROR.equals(status)) {
```

Comment:
you can compare enums with `==` directly (there is an intellij setting to mark these and auto-fix them for you)
```suggestion
        if (HttpStatus.INTERNAL_SERVER_ERROR == status) {
```